README.md

```diff
@@ -250,6 +296,7 @@ For multi-select inputs, when providing a custom `filterOptions` method, remembe
 	optionRenderer	|	func	|	undefined	|	function which returns a custom way to render the options in the menu
 	options 	|	array	|	undefined	|	array of options
 	placeholder 	|	string\|node	|	'Select ...'	|	field placeholder, displayed when there's no value
+	renderMenu | func | undefined | Renders a custom menu with options; accepts the following named parameters: `renderMenu({ focusedOption, focusOption, options, selectValue, valueArray })`
```

Comment:
maybe `menuRenderer` would be better choice since it follows existing naming convention of `optionRenderer `